### DISCUSSION TITLE
Donwload dos videos comprados

### DISCUSSION THREAD

**Discussion by cmykel** [heliomarpm/udemy-downloader-gui#96] [Q&A] [answered]
Olá, percebi que o programa apenas permite baixar os cursos comprados, caso vc tenha uma assinatura individual os cursos que você adicionou a uma lista para cursar não aparecem para baixar, existe a possibilidade de futuramente isso acontecer?

**Comment by heliomarpm** [comment, marked as answer]
> Olá, percebi que o programa apenas permite baixar os cursos comprados, caso vc tenha uma assinatura individual os cursos que você adicionou a uma lista para cursar não aparecem para baixar, existe a possibilidade de futuramente isso acontecer?

Desculpe pela demora, não costumo acompanhar, as discussões.

Bom, respondendo a pergunta, o fato de criar uma lista no Udemy não quer dizer que o curso foi comprando, mas qualquer curso dessa lista ou fora dela você vai ter como baixar se tiver sido comprado.
Vale lembrar que cursos quando estão protegidos pela criptografia DRM, não é possível baixar pelo app.
